### ISSUE TITLE
Ongoing location sharing

### ISSUE THREAD

**Issue by gsdgethjfjj** [organicmaps/organicmaps#7697]
**Is your feature request related to a problem? Please describe.**

For now, I can only share my location with a link that needs to be sent to another application. This link contains coordinates without real time location updates.

**Describe the ideal solution**

I would like to be able to share my location in real time using Organic Maps with a setting of duration (hour, day, until turned off). It would also be great to include notifications of arrival and departure from the selected locations.

**Describe alternatives you have considered**

Now I have to use Google Maps to share location in real time. Here is a video showing how to set up this function.
https://youtube.com/watch?v=5amIspnlY7I

**Additional context**

Unfortunately Google (and Apple) are notorious for their bad privacy policies. Organic Maps is positioned as privacy-respecting offline maps. Adding one online feature will increase end-user privacy because only convenient and easy alternatives are Apple Maps and Google Maps as far as I know.

**Comment by heretovote** [comment]
Yes, this would be amazing. There is no open source alternative.
It could be a paid option for the server costs / support as well.

**Comment by chozabu** [comment]
A way to share location in realtime automatically with selected contacts in organic maps would be fantastic.
Bonus points if it can be for a limited time, or single ping.
Massive bonus points if it can be a decentralised protocol!
